src/instrumentations/https/HttpsInstrumentation.ts

```diff
@@ -0,0 +1,7 @@
+import LumigoHttpInstrumentation from './HttpInstrumentation';
```

Comment:
i dont like spliting it to Http and Https since its the same intrumentor and just add complexity 
I would just make it http